apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -172,6 +172,45 @@ export const updatePageById = (
     .executeTakeFirstOrThrow()
 }
 
+export const getBlobOfResource = async (
+  tx: Transaction<DB>,
+  resourceId: string,
+) => {
```

Comment:
question - should we use named argument instead of positional argument?

Reply:
i usually am ok with either if there's no confusion - in this case, the first argument has very different typing from the second, so i used positional. 

i can update it to a named arg!

Reply:
okay thanks for explaining! no issue with this. just thought that in general positional argument are easier to use as an API interace too